src/client/method-list-units.c

```diff
@@ -149,3 +151,16 @@ void print_unit_list_simple(UnitList *unit_list, const char *glob_filter) {
                 }
         }
 }
+
+int method_list_units(Command *command, void *userdata) {
+        Client *client = (Client *) userdata;
+        int r = 0;
+        char *filter_glob = command_get_option(command, ARG_FILTER_SHORT);
+        if (command->opargc == 0) {
+                r = method_list_units_on_all(client->api_bus, print_unit_list_simple, filter_glob);
+                return r;
+        }
+
+        char *node_name = command->opargv[0];
+        return method_list_units_on(client->api_bus, node_name, print_unit_list_simple, filter_glob);
```

Comment:
Do you really need `node_name`? How about:
```suggestion
        return method_list_units_on(client->api_bus, command->opargv[0], print_unit_list_simple, filter_glob);
```

Reply:
Ack